model/registry/gen-ai.yaml

```diff
@@ -148,3 +148,8 @@ groups:
           If one of the predefined values applies, but specific system uses a different name it's RECOMMENDED to document it in the semantic
           conventions for specific GenAI system and use system-specific name in the instrumentation.
           If a different name is not documented, instrumentation libraries SHOULD use applicable predefined value.
+      - id: gen_ai.system_fingerprint
+        stability: experimental
+        type: string
+        brief: A fingerprint to track any eventual change in the Generative AI environment.
```

Comment:
Not sure about this description, for reference the instrumentation that requires this field used `System fingerprint of the system that generated the response`